### PR TITLE
feat: add loading props

### PR DESCRIPTION
**Pull request by chenshuai2144**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

pageHeader will use a period of time when using asynchronous data, loading can solve this problem

### 💡 Solution

image

### 📝 Changelog description

PageHeader add new props `loading`.
PageHeader 支持 `loading` props。

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
feature branch

File: components/page-header/index.tsx [hunk 1]
```diff
@@ -60,14 +62,17 @@ const renderHeader = (prefixCls: string, props: PageHeaderProps) => {
...
+    return (
+      <div className={`${prefixCls}-title-view`}>
+        <span className={`${titlePrefixCls}-title`}>{title}</span>
+        {subTitle && <span className={`${titlePrefixCls}-sub-title`}>{subTitle}</span>}
+        {tags && <span className={`${titlePrefixCls}-tags`}>{tags}</span>}
+        {extra && <span className={`${titlePrefixCls}-extra`}>{extra}</span>}
+      </div>
+    );
+  }
+  return null;
```
Comment (afc163) [inline components/page-header/index.tsx]: 有头像的 PageHeader 可以改成有头像的 Skeleton

Comment (chenshuai2144) [reply components/page-header/index.tsx]: PageHeader 的 现在没有头像的配置。
pro 中 PageHeader 带了一个头像，改成一致的？

File: components/page-header/index.tsx [hunk 2]
```diff
@@ -98,12 +104,14 @@ const PageHeader: React.SFC<PageHeaderProps> = props => (
       );
 
       return (
-        <div className={className} style={style}>
-          {renderHeader(prefixCls, props)}
-          {renderTitle(prefixCls, props)}
-          {children && <div className={`${prefixCls}-content-view`}>{children}</div>}
-          {renderFooter(prefixCls, footer)}
-        </div>
+        <Skeleton loading={loading} active paragraph={{ rows: 2 }}>
```
Comment (zombieJ) [inline components/page-header/index.tsx]: 这个 Skeleton 包在最外面，岂不就和
``` jsx
{loading ? (
  <Skeleton {...props} />
) : (
  <PageHeader {...props} />
)}
```

一样的？

至少搞在 PageHeader 样式下，否则带背景是这样的：
屏幕快照 2019-04-01 下午1 30 19

File: components/page-header/style/index.less [hunk 3]
```diff
@@ -23,6 +23,11 @@
     i:hover {
       color: @primary-color;
     }
+    .ant-skeleton {
```
Comment (zombieJ) [inline components/page-header/style/index.less]: 用变量……

**Comment by chenshuai2144** [comment]
image

**Comment by chenshuai2144** [comment]
image

**Comment by chenshuai2144** [comment]
用Paragraph包裹一下？
https://ant.design/components/typography/

**Pull request closed without merging**